Buildings/Fluid/DXSystems/Heating/AirSource/Validation/BaseClasses/PLRToPulse.mo

```diff
@@ -2,9 +2,16 @@ within Buildings.Fluid.DXSystems.Heating.AirSource.Validation.BaseClasses;
 block PLRToPulse
```

Comment:
This block's implementation may need to be improved. As it will cause some state events when simulating with Optimica and the simulation hangs.